compiler/rustc_middle/src/ty/structural_impls.rs

```diff
@@ -177,7 +177,10 @@ impl<'tcx> fmt::Debug for ty::PredicateKind<'tcx> {
                 write!(f, "TypeWellFormedFromEnv({:?})", ty)
             }
             ty::PredicateKind::Ambiguous => write!(f, "Ambiguous"),
-            ty::PredicateKind::AliasEq(t1, t2) => write!(f, "AliasEq({t1:?}, {t2:?})"),
+            // TODO
```

Comment:
I thought we blocked CI on there not being any `TODO` in rustc :( why is CI passing. This looks correct to me anyway, this `// TODO` should probably be removed.

Reply:
oh, yeah, it should be failing tidy, lol